### PR TITLE
loadout tweaks

### PR DESCRIPTION
**Pull request by cebutris**
some changes to loadouts for some nerds

### CODE REVIEW

**Review by Tk420634 — APPROVED** [review]
denied, wolf

File: modular_citadel/code/modules/client/loadout/__donator.dm [hunk 1]
```diff
@@ -514,13 +514,9 @@
...
-	new /obj/item/gun/ballistic/automatic/pistol/ninemil/ruby(src)
-	new /obj/item/ammo_box/magazine/m9mm(src)
-	new /obj/item/ammo_box/c9mm(src)
-	new /obj/item/gun/ballistic/revolver/shotpistol(src)
-	new /obj/item/ammo_box/shotgun/buck(src)
-	new /obj/item/melee/onehanded/knife/switchblade(src)
-	new /obj/item/storage/survivalkit/combat/empty(src)
+	new /obj/item/screwdriver/nuke/nt(src)
+	new /obj/item/gun_upgrade/trigger/raidertrigger(src)
+	new /obj/item/storage/survivalkit/combat(src)
```
Comment (cebutris) [inline modular_citadel/code/modules/client/loadout/__donator.dm:519]: Whoops how did this happen. Gimme a sec I need to fix it!!

**Comment by cebutris** [comment]
OKAY I don't know how those errors snuck in but they're out now. Should be ready!

**Pull request merged**